dependencies.list

```diff
@@ -1,4 +1,4 @@
 VERSION=2.4.3
-REALM_CORE_VERSION=2.3.1
-REALM_SYNC_VERSION=1.1.0
+REALM_CORE_VERSION=2.3.2
+REALM_SYNC_VERSION=1.3.0
```

Comment:
It actually looks like there are no public-facing changes for sync, and the core changes are both for Android issues.